mantidproject/mantid
# `SaveFocusedXYE` to write TEMP into the header

This PR modifies slightly how the saver writes the header part of the file for XYE format (fullprof number 10).
The request was to add the metadata value (most commonly temperature) corresponding to the diffractogram into the file. 
For temperature in particular, there exist a keyword `TEMP` that is parsed by fullprof, but only if the file starts with `XYDATA` keyword.
For the rest, the metadata information line should be commented with #.
Since the comment lines are restricted to 6, the other lines are squeezed to free a room for the new entries.
The test is modified accordingly.
See the link in the issue for more information.

**To test:**
This should be tested by someone who is familiar with XYE exports and has fullprof installed.
Make sure that your exports with this change are still readable by fullprof.



Fixes #20940 . 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
